### PR TITLE
[Coinmate] Rename package for consistency with the rest

### PR DESCRIPTION
**Pull request by makarid**
and changing nonce type from Long to Synchronised<Long> in PusherAuthParamsObject.

### CODE REVIEW

File: xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/PusherAuthParamsObject.java [hunk 1]
```diff
@@ -15,21 +18,33 @@
 
 public class PusherAuthParamsObject {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(PusherAuthParamsObject.class);
```
Comment (badgerwithagun) [inline xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/PusherAuthParamsObject.java]: `LOGGER` is the correct convention for a `static final`.

File: xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/CoinmateUrlEncodedConnectionFactory.java [hunk 2]
```diff
@@ -24,21 +27,21 @@ public String getContentType() {
...
             while(var2.hasNext()) {
                 String parameterName = (String)var2.next();
                 urlParameters.append("&").append(parameterName).append("=");
-                urlParameters.append(URLEncoder.encode((String)mQueryStringParameters.get(parameterName), this.getCharset()));
+                urlParameters.append(URLEncoder.encode((String)mQueryStringParameters.get(parameterName), getCharset()));
             }
-        } catch (UnsupportedEncodingException var4) {
-            var4.printStackTrace();
+        } catch (IOException e) {
+            log.error(e.getMessage(),e);
```
Comment (badgerwithagun) [inline xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/CoinmateUrlEncodedConnectionFactory.java]: Any reason why this doesn't just rethrow?

File: xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/PusherAuthParamsObject.java (hunk 1 above)
Comment (makarid) [reply xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/PusherAuthParamsObject.java]: yes, i put the log in order to be consistent with @Log tag on lombok. I can change it if you like

File: xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/CoinmateUrlEncodedConnectionFactory.java (hunk 2 above)
Comment (makarid) [reply xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/CoinmateUrlEncodedConnectionFactory.java]: Rethrow without log.error(e.getMessage(),e)?

Comment (badgerwithagun) [reply xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/CoinmateUrlEncodedConnectionFactory.java]: Yeah - I mean the exception will get caught somewhere. Seems odd to squash it.

Comment (makarid) [reply xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/CoinmateUrlEncodedConnectionFactory.java]: OK, i delete the log and now it just throws new CoinmateException

**Pull request merged**